templates/availability/tab-display-options.phtml

```diff
@@ -3,12 +3,10 @@
         Calendar and timezone options
     </div>
     <div class="wide-form__row">
-        <div class="wide-form__label">
-            Show dates and times on the site using the customer's timezone
-        </div>
-        <div class="wide-form__input">
-            <bool-switcher v-model="useCustomerTimezone"/>
+        <div class="wide-form__label" style="width: 100%;max-width: 600px">
+            Dates and times are shown to the customer using their browser's timezone, both during the booking process and at checkout. Soon, this will be optional.
         </div>
+        <?php //<div class="wide-form__input"> <bool-switcher v-model="useCustomerTimezone"/> </div> ?>

```

Comment:
This is here because this change is temporary?

Reply:
Yes